lib/laborApproval.js

```diff
@@ -0,0 +1,107 @@
+const ServiceBase = require('./serviceBase');
+const SimpleSchema = require('simpl-schema').default;
+const moment = require('moment');
+
+const SERVICE = 'Erp.Bo.LaborApprovalSvc';
+
+// Build a [where, whereDtl] condition based on some pre-defined fields
+const buildWhere = criteria => {
```

Comment:
Can we share this one with the one in Labor.js, or is there a subtle difference between them?